src/booking/containers/booking-date/booking-date.component.scss

```diff
@@ -0,0 +1,6 @@
+/deep/.date-head h1 {
+  margin-top: 0px;
+  margin-bottom: 10px;
+  font-weight: bold ;
```

Comment:
Superfluous space between bold and ;

Reply:
Hi @paulhowes-HMCTS - resolving on your behalf since @tarikdogan-hmcts has removed the SCSS file itself (it's no longer required).